src/api/public/apidocs-new/paths/build_project.yaml

```
@@ -0,0 +1,87 @@
+get:
+  summary: Get a simple directory listing of all repositories for the specified project
+  description: Get a simple directory listing of all repositories for the specified project
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+  responses:
+    '200':
+      description: |
+        OK
+
+        XML Schema used for body validation: [directory.xsd](../schema/directory.xsd)
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/directory.yaml'
+          example:
+            entry:
+              - name: 'openSUSE_Tumbleweed'
+              - name: 'openSUSE_Leap_15.3'
+    '401':
+      $ref: '../components/responses/unauthorized.yaml'
```

We can also get a `404` when the `project_name` doesn't correspond to an existing one.

```suggestion
    '401':
      $ref: '../components/responses/unauthorized.yaml'
    '404':
      $ref: '../components/responses/unknown_project.yaml'
```

---

@saraycp done